xml/depl_conf_admin_crowbar.xml

```diff
@@ -1989,16 +1989,15 @@ storage: eth0
       <title>Setting Up Multiple External Networks</title>
    <para>
 This section shows you how to create external networks
-on &cloud; by modifying the &o_netw; &barcl;.
+on &cloud; by modifying the Network and &o_netw; &barcl;s.
    </para>
 
  <sect2 xml:id="sec.config.multi.ext.networks">
-     <title>Configure the &o_netw; &Barcl;</title>
+     <title>Configure the &Barcl;s</title>
      <procedure>
          <step>
 <para>
-    Add the following lines to
-    <filename>/etc/crowbar/network.json</filename> to set up an external network, using the name of your new network, VLAN ID, and network addresses.
+    First, modify the Network &barcl; to configure your external network, using the name of your new network, VLAN ID, and network addresses.
```

Comment:
this para will have a `1.` prepended to it, so the "First," seems a bit superfluous here